dart/biomechanics/MarkerFitter.cpp

```diff
@@ -4588,22 +4602,25 @@ ScaleAndFitResult MarkerFitter::scaleAndFit(
 #endif
     }
 
+    const int outputDim
+        = (markerObservations.size() * 3) + (joints.size() * 3) + 1;
```

Comment:
What is the additional dimension here for?

Reply:
I'm pushing a comment in the code, but it's there for the anthropometric prior cost term.